libraries/JGUZDV.Blazor.Components/src/Modals/Modal.razor

```diff
@@ -50,11 +50,17 @@
     [Parameter]
     public string? ModalFooterClasses { get; set; }
 
-    protected override async Task OnParametersSetAsync()
+    protected override async Task OnInitializedAsync()
```

Comment:
Modale haben zwar ohnehin schon ein Problem mit dem "neuen" Rendermodus in Blazor, aber wir müssen es nicht schlimmer machen - lest mal über JSRuntime und Blazor. Da sollte sowas rauskommen, wie "macht das in OnAfterRender()".